tfs/src/main/java/hudson/plugins/tfs/TeamPullRequestMergedDetailsAction.java

```diff
@@ -24,7 +24,7 @@
     private static final long serialVersionUID = 1L;
     private static final String URL_NAME = "team-pullRequestMergedDetails";
 
-    public GitPullRequestEx gitPullRequest;
+    public transient GitPullRequestEx gitPullRequest;

```

Comment:
GitPullRequestEx is not serializable so in case someone was serializing it, it would have already broken.